R/stattle.R

```diff
@@ -14,9 +13,8 @@
 #' 
 #' @examples 
 #' \dontrun{
-#' TOKEN <- "insert-your-token-here"
-#' results <- stattle(TOKEN, 
-#'                    sport="hockey", 
+#' setToken("insert-your-token-here")
+#' results <- stattle(sport="hockey", 
```

Comment:
Most R database packages have functions called something like `dbGetQuery()`. We could do `ssGetResult()` or something.  It's also strange that the [Google Style Guide](https://google.github.io/styleguide/Rguide.xml#functiondefinition) recommends using capital letters for the first letter of function names but I've rarely seen function names being capitalized.